source/administration/concepts.rst

```diff
@@ -163,3 +163,7 @@ MinIO provides a configuration option to create buckets with versioning enabled.
 When enabled, MinIO writes mutated objects to a different version than the original, allowing access to both the original object and the newer, changed object.
 
 Additional configurations on the MinIO bucket determine how long to retain older versions of each object in the bucket.
+
```

Comment:
Perhaps in the backup and restore section? To dissuade people from thinking that copying files on disk will be anything other than a world of sadness.

Reply:
Yeah, that makes more sense. Thanks for the suggestion.